src/config/express.js

```diff
@@ -0,0 +1,60 @@
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const env = require('./env');
+const favicon = require('serve-favicon');
+const bodyParser = require('body-parser');
+const session = require('express-session');
+const cookieParser = require('cookie-parser');
+const BunyanMiddleware = require('bunyan-middleware');
+
+module.exports = logger => {
+    /**
+     * Global App Config
+     */
+    let app = express();
+    app.use(BunyanMiddleware({
+        headerName: 'X-Request-Id',
+        propertyName: 'reqId',
+        logName: 'req_id',
+        obscureHeaders: [],
+        logger: logger
+    }));
+    app.use(session({
+        saveUninitialized: true,
+        resave: true,
+        secret: env('SESSION_SECRET')
+    }));
+    app.use(bodyParser.json());
```

Comment:
Responding in xml is something that we could add later. We could use [object-to-xml](https://www.npmjs.com/package/object-to-xml), for example. But I think this is not a concern right now.